client/src/components/UserTrainingPlans.jsx

```diff
@@ -1,24 +1,79 @@
 //Import necessary functionalities
-import React, { useContext, useState } from 'react';
-import ComponentContext from '../ComponentContext.js';
+import React, { useState, useEffect } from 'react';
+import { UserStatus } from '../context/UserContext';
+import { useNavigate } from 'react-router-dom';
 
-const UserTrainingPlans = () => {
+const UserTrainingPlans = ({ selectedPet }) => {
+    
+    //Use useNavigate hook from React Router to establish functionality in buttons below
+    const navigate = useNavigate();
+    
+    //Import user settings to use in data pulls
+    const { loggedUser, setLoggedUser } = UserStatus();
     
-    const { setComponent } = useContext(ComponentContext);
-
-    //Will need to pull all training plan information for this particular user via server GET request
     //Hold training plan info in state
-    const [trainingPlans, setTrainingPlans] = useState([]);
+    const [trainingPlans, setTrainingPlans] = useState(null);
```

Comment:
Could use a comment here explaining why you are passing in null for useState. What would the value be if you didn't pass in null?

Reply:
Just added that through commit [97a0c0c](https://github.com/tati-guzman/click-and-treat/pull/3/commits/97a0c0c513881754a684e221ecb8dcc1bab926ee)! 

The empty array was being evaluated as truthy in my petDisplay function and I needed "Please select pet" to be displayed until there is data in this state. Rather than have it check for length of the array, I decided to have it check for the presence of any values! That is written out now in the comments. Thanks for pointing it out!